readme.md

```diff
@@ -4,6 +4,10 @@ Providing you with all sorts of Good Eggs-style fake data!
 
 All exported functions are stateless (meaning you don't have to worry about binding to `this`).
 
+## Purpose and Scope
+
+Use `fake-eggs` to generate all random factory data except for json schemas where [unionized](https://github.com/goodeggs/unionized) should be employed.
+
```

Comment:
We had discussed whether to bless fake-eggs for non-test use cases where random realistic data is necessary (e.g. sanitizing a dev database, populating mock data for local development, etc.). Should we add a few words here calling out that its intended scope isn't limited to tests? It already doesn't technically _say_ it's limited to tests, but I can see it being assumed to be implied by some readers.

Reply:
What do you think of this line? Encouraging enough for the use cases beyond test data? 

> This module can be used for test data, local development, and beyond!